cmd/kaleido-go.go

```diff
@@ -28,13 +28,11 @@ func initLogging(debugLevel int) {
 	switch debugLevel {
 	case 0:
 		log.SetLevel(log.ErrorLevel)
-		break
 	case 1:
 		log.SetLevel(log.InfoLevel)
-		break
 	default:
+		debugLevel = 2
```

Comment:
I think the right default is `INFO` (and the default is set elsewhere int he `cobra config`), but if you think there are critical messages missing we should get those added at info-level